jwst/lib/set_telescope_pointing.py

```diff
@@ -1110,54 +1104,91 @@ def _roll_angle_from_matrix(matrix, v2, v3):
     return new_roll
 
 
-def _get_vertices_idl(aperture_name, useafter, prd_db_filepath=None):
+def _read_wcs_from_siaf(aperture_name, useafter, prd_db_filepath=None):
+    """
+    Read WCS keywords from the SIAF database file.
```

Comment:
I'm not sure I would characterize this as reading "WCS keywords" from the SIAF. We're getting values from the SIAF that are then used to populate FITS WCS keywords in the science data. There aren't any actual WCS keywords in the SIAF (but it's only a doc string, not working code, so ...)